htdocs/societe/index.php

```diff
@@ -367,7 +367,7 @@
 $boxlist .= '</div>'."\n";
 
 $boxlist .= '<div class="secondcolumn fichehalfright boxhalfright" id="boxhalfright">';
-$boxlist .= $lastmodified;
+$boxlist .= empty($lastmodified) ? '' : $lastmodified;
```

Comment:
$lastmodified  is previously set and sometimes not.  
A better fix would be to initialise to '' at top of file so it is always defined and 
$boxlist .= $lastmodified;
will be enough.